src/main/java/hudson/plugins/analysis/graph/GraphConfiguration.java

```diff
@@ -433,6 +493,8 @@ private void reset() {
         dayCount = DEFAULT_DAY_COUNT;
         graphType = DEFAULT_GRAPH;
         useBuildDate = DEFAULT_USE_BUILD_DATE;
+        parameterName = null;
```

Comment:
I don't understand this comment. 

I generally think it is safer to use empty strings rather than null values to avoid NPE. If this is not possible, then using null is ok.